pkg/components/exporter/exporter.go

```diff
@@ -48,8 +48,16 @@ const (
 	DaemonSetName = prefix + "ds"
 	ServiceName   = prefix + "svc"
 
-	// TODO: make this config to the operator
-	imageRepo = "quay.io/sustainable_computing_io/kepler:"
+	StableImage = "quay.io/sustainable_computing_io/kepler:release-0.5.4"
```

Comment:
Does this mean we will go with `release-0.5.4` for dev-preview ?

Reply:
It depends on the outcome of the testing :)